extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -50,7 +50,10 @@ public static function parserTagInit( Parser $parser ) {
 	 * @return string
 	 */
 	public static function replaceInfoboxMarkers( &$parser, &$text ) {
-		$text = static::getInstance()->replaceMarkers( $text );
+		global $wgArticleAsJson;
+		if ( !$wgArticleAsJson ) {
```

Comment:
Please add the info that it's done in `PortableInfoboxHooks::onArticleAsJsonBeforeEncode` for Mercury.